pnpm-lock.yaml

```diff
@@ -1665,6 +1665,34 @@ importers:
         specifier: 10.0.0
         version: 10.0.0
 
+  packages/xml-odata-annotation-converter:
+    dependencies:
+      '@sap-ux/odata-annotation-core':
+        specifier: workspace:*
+        version: link:../odata-annotation-core
+    devDependencies:
+      '@sap-ux/odata-annotation-core-types':
+        specifier: workspace:*
+        version: link:../odata-annotation-core-types
+      '@sap-ux/odata-vocabularies':
+        specifier: workspace:*
+        version: link:../odata-vocabularies
+      '@xml-tools/ast':
+        specifier: 5.0.5
+        version: 5.0.5
+      '@xml-tools/parser':
+        specifier: 1.0.11
+        version: 1.0.11
+      chevrotain:
+        specifier: 7.1.1
+        version: 7.1.1
+      npm-run-all:
+        specifier: 4.1.5
+        version: 4.1.5
+      typescript:
+        specifier: 4.9.5
```

Comment:
@donal-tobin-sap  Currently 4.x version is used in this repository https://github.com/SAP/open-ux-tools/blob/main/package.json#L31. I assume that we'd want to update the version at the same time for the whole repository instead of doing it for individual packages.